### PR TITLE
scheduler: allow 1 GC command at the same time.

### PR DESCRIPTION
**Pull request by disksing**
@siddontang @zhangjinpeng1987

### CODE REVIEW

File: src/storage/mod.rs [hunk 1]
```diff
@@ -960,6 +960,20 @@ mod tests {
...
 
     #[test]
+    fn test_sched_1gc() {
+        let config = Config::new();
+        let mut storage = Storage::new(&config).unwrap();
+        storage.start(&config).unwrap();
+        let (tx1, rx1) = channel();
+        storage.async_gc(Context::new(), 1, expect_ok(tx1.clone())).unwrap();
+        let (tx2, rx2) = channel();
+        storage.async_gc(Context::new(), 1, expect_too_busy(tx2.clone())).unwrap();
```
Comment (zhangjinpeng1987) [inline src/storage/mod.rs]: The first gc command maybe finished before the second gc command send.

**Comment by zhangjinpeng1987** [comment]
rest LGTM

File: src/storage/txn/scheduler.rs [hunk 2]
```diff
@@ -506,6 +509,9 @@ impl Scheduler {
     }
 
     fn insert_ctx(&mut self, ctx: RunningCtx) {
+        if ctx.tag == "gc" {
```
Comment (siddontang) [inline src/storage/txn/scheduler.rs]: use const name for "gc"

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
CI failed, please fix it ASAP

**Pull request merged**